src/plugins/irc-events/error.js

```diff
@@ -8,8 +8,14 @@ module.exports = function(irc, network) {
 	irc.on("irc error", function(data) {
 		let text = data.error;
```

Comment:
This probably needs to be an empty string, if there is no `data.channel`, the formatted error will look odd.